### PR TITLE
Create rollups from scratch instead of retrieving them from L1 blocks

### PR DESCRIPTION
**Pull request by joeldudley**
### Why is this change needed?

- Provide a description and a link to the underlying ticket

### What changes were made as part of this PR:

- Provide a high level list of the changes made
- List key areas for the reviewer 

### :rotating_light: Definition of done :rotating_light:
- [ ] Unit tests added to cover new or changed functionality 
- [ ] Docs pages updated to cover new or changed functionality
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

**Review by BedrockSquirrel — APPROVED** [review]
LGTM - would be good to have the build passing though, else it's unusable for everybody

File: go/host/host.go [hunk 1]
```diff
@@ -876,40 +882,47 @@ func (h *host) checkBlockForSecretResponse(block *types.Block) bool {
...
-				return fmt.Errorf("could not create batch request. Cause: %w", err)
-			}
+		isParentStored, batchRequest, err := h.batchManager.IsParentStored(batch)
+		if err != nil {
+			return fmt.Errorf("could not determine whether batch parent was missing. Cause: %w", err)
+		}
+
+		// We have encountered a missing parent batch. We abort the storage operation and request the missing batches.
+		// We only do this if the batches did not themselves arrive as part of catch-up, to avoid excessive P2P pressure.
+		if !isParentStored && !batchMsg.IsCatchUp {
```
Comment (BedrockSquirrel) [inline go/host/host.go:912]: Don't you need to also handle the case where `!isParentStored` but `isCatchup`? As it stands you send it to the enclave to fail I think.

**Review by BedrockSquirrel — APPROVED** [review]
LGTM, minor question

File: go/host/host.go (hunk 1 above)
Comment (joeldudley) [reply go/host/host.go]: Fixed.

**Pull request merged**